packages/shared/src/components/cards/squad/UnfeaturedSquadGrid.tsx

```diff
@@ -10,39 +10,44 @@ import {
 } from '../../typography/Typography';
 import { largeNumberFormat } from '../../../lib';
 import { Separator } from '../common';
-import { SquadImage } from './common/SquadImage';
-import { SquadJoinButtonWrapper } from './common/SquadJoinButton';
 import { UnFeaturedSquadCardProps } from './common/types';
+import { Origin } from '../../../lib/log';
+import { SquadJoinButton } from '../../squads/SquadJoinButton';
+import { ButtonVariant } from '../../buttons/common';
 
 export const UnfeaturedSquadGrid = ({
   source,
-  title,
-  subtitle,
-  icon,
-  action,
-  description,
+  className,
 }: UnFeaturedSquadCardProps): ReactElement => {
+  const router = useRouter();
+  const title = source.name;
+
   return (
-    <Card className={classNames('min-w-[19rem] overflow-hidden border-0 p-4')}>
+    <Card
+      className={classNames(
+        'w-[19rem] overflow-hidden border-0 p-4',
+        className,
+      )}
+    >
       <CardLink
         href={source.permalink}
         rel="noopener"
         title={source.description}
       />
       <div className="mb-3 flex items-center justify-between">
-        <SquadImage
-          image={source?.image}
-          icon={icon}
-          title={title}
-          size="size-16"
+        <img
```

Comment:
Shouldn't we use `Image` here?

Reply:
Right. Yeah, let's use it wherever we can.